app/server/models/user.py

```diff
@@ -293,12 +297,22 @@ def update_last_seen_ts(self):
         else:
             self._last_seen = cur_time
 
+    @staticmethod
+    def salt_hash_password(password):
```

Comment:
another renaming one (sorry lol), but perhaps rename to 'salt_hash_secret' and change the `password` fields to `secret` has well, given it now hashes both the pin and the password. (Same with check/verify below)